### PR TITLE
docs/ppdse 2340 update space page

### PR DESCRIPTION
**Pull request by baburay23**
PPDSC-2340

**What**

1. Background - updated spacing page removed spaceinset spaceStack and added logical props
2. What did you do
3. What does the reviewers should expect

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdse-2340-update-spcae-pages
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdse-2340-update-spcae-pages/storybook

**Review by Xin00163 — COMMENTED** [review]
Looks good. I noticed in the design spec that `spaceInset` & `spaceStack` are removed. We haven't deprecated them yet. Is it OK to remove from docs?

**Review by Xin00163 — APPROVED** [review]

**Comment by LukeFinch** [comment]
Hi Aysha,
In the designs, it looks as though we're purposefully adding line breaks in the paragraphs to stop the line lengths from running on for too long. 

Perhaps we could try adding a max-width to the text blocks so that they don't go too wide on larger viewports?
Something like `max-width: 60ch` - or perhaps a number of column spans? 

I'd consult with the design team as it looks intentional on their end.

File: site/pages/theme/foundation/spacing.tsx [hunk 1]
```diff
@@ -201,15 +78,13 @@ const Spacing = (layoutProps: LayoutProps) => (
           <Illustration path="theme/spacing/overview" />
         </ContentPrimary>
 
-        <ContentSecondary
+        <ContentPrimary
+          headline="Space Tokens"
```
Comment (LukeFinch) [inline site/pages/theme/foundation/spacing.tsx:82]: Lower case T in designs. `Space tokens` versus `Space Tokens`

**Review by LukeFinch — CHANGES_REQUESTED** [review]

**Comment by LukeFinch** [comment]
> 

https://nidigitalsolutions.jira.com/browse/PPDSC-2535
Mike's got a ticket in the backlog saying the layout of these pages has regressed.

**Review by LukeFinch — APPROVED** [review]

**Pull request merged**